components/home/Home.jsx

```diff
@@ -1,85 +1,68 @@
 import * as React from 'react';
-import { View, StyleSheet } from 'react-native';
-import { Button, Title } from 'react-native-paper';
-import WorkoutContainer from './WorkoutContainer';
+import {
+  SafeAreaView, Image,
+} from 'react-native';
+import styled from 'styled-components/native';
+import CurrentCycle from './CurrentCycle';
+import WorkoutList from './WorkoutList';
 
-export default () => (
-  <View style={styles.container}>
+const Title = styled.Text`
+  font-family: 'Montserrat_700Bold';
+  font-size: 40px;
+  margin: 0px 6%;
+`;
 
-    {/* Cycles Section */}
-    <View style={{ ...styles.sectionComponent, margin: 30 }}>
+const welcomeName = 'Justin';
 
-      <Title style={styles.title}>
-        Cycles
-      </Title>
-
-      <Button contentStyle={{ width: 331, height: 170 }} style={styles.startCycle} color="#FFFFFF" onPress={() => alert('This should start the current workout in the cycle')}>
-        Start Cycle Workout
-      </Button>
-
-      <Button contentStyle={{ width: 325, height: 35 }} style={styles.seeAllButton} color="#000000" onPress={() => alert('This should take you to workouts tab')}>
-        See All Cycles
-      </Button>
-
-    </View>
-
-    {/* Workouts Section */}
-    <View style={{ ...styles.sectionComponent, margin: 10 }}>
-      <Title style={styles.title}>
-        Workouts
-      </Title>
-
-      <WorkoutContainer
-        items={[{ name: 'Push Workout A' }, { name: 'Pull Workout A' }]}
+export default () => {
+  const items = [
+    {
+      name: 'Back & Biceps', subtext: '1 day', color: '#CAB0FF', onPress: () => alert('Push, Pull, Legs A'),
+    },
+    {
+      name: 'Upper A RP', subtext: '7 days', color: '#9D8DFF', onPress: () => alert('Bro Split A'),
+    },
+    {
+      name: 'Legs A', subtext: '1 day', color: '#6D8DFF', onPress: () => alert('Upper Lower Split A'),
+    },
+    {
+      name: 'Pull B', subtext: '2 days', color: '#CAB0FF', onPress: () => alert('Push, Pull, Legs B'),
+    },
+    {
+      name: 'Push B', subtext: '1 day', color: '#9D8DFF', onPress: () => alert('Bro Split B'),
+    },
+    {
+      name: 'Legs B', subtext: '22 days', color: '#6D8DFF', onPress: () => alert('Upper Lower Split B'), onIconPress: () => alert('Edit Upper Lower Split B'),
+    },
+  ];
+  const cycleDetails = {
+    name: 'Pull A', subtext: 'Back, Biceps', color: '#CAB0FF', onPress: () => alert('Start Pull A'),
+  };
+  return (
+    <SafeAreaView>
+      <Image
+        source={require('../../assets/logo.png')}
+        style={{
+          width: 215, height: 215, position: 'absolute', right: 10, top: 20,
+        }}
       />
+      <SafeAreaView style={{ marginBottom: '10%', marginTop: '5%' }}>
+        <Title>Hello</Title>
+        <Title>
+          {welcomeName}
+          !
+        </Title>
+      </SafeAreaView>
+      <SafeAreaView style={{ height: '50%', marginBottom: '25%' }}>
```

Comment:
Add margin left and margin right to this, also I think this should be just a view not a SafeAreaView (@ctperry0301?)